scripts/drtprod

```diff
@@ -240,7 +210,17 @@ EOF"
         ;;
     esac
 
-    roachprod ssh ${cluster} -- 'sudo systemctl enable datadog-agent && sudo systemctl restart datadog-agent'
+    dd_api_key="$(gcloud --project=cockroach-drt secrets versions access latest --secret datadog-api-key)"
+    if [ -z "${dd_api_key}" ]; then
+      echo "Missing Datadog API key!"
+      exit 1
+    fi
+
+    roachprod ssh ${cluster} -- "sudo tee /etc/profile.d/99-datadog.sh > /dev/null << EOF
+export DD_SITE=us5.datadoghq.com
+export DD_API_KEY=${dd_api_key}
+export DD_TAGS=env:development,cluster${cluster%:*},team:drt,service:drt-cockroachdb
+EOF"
```

Comment:
With this configuration operators won't have to pass any arguments to `roachtest` since it'll respect these environment variables.

Reply:
I'm actually not sure if we wanna throw these into `profile.d`. As convenient as that sounds, it might just be best to let us add these manually to `roachtest_ops.sh` in the workload node. We should be moving the creation of that script into this script though, as currently it's less obvious.

Reply:
I'm not familiar with `roachtest_ops.sh`. I'll take a stab at moving the logic into that location instead. Thanks for letting me know about it.

Reply:
In the interest of time I left a TODO for this.